src/equations/2d/ideal_glm_mhd.jl

```diff
@@ -415,14 +415,21 @@ function flux_hll(u_ll, u_rr, orientation, equation::IdealGlmMhdEquations2D)
   return SVector(f1, f2, f3, f4, f5, f6, f7, f8, f9)
 end
 
-
-# strong form of nonconservative flux on a side, e.g., the Powell term
-#     phi^L 1/2 (B^L+B^R) normal - phi^L B^L normal = phi^L 1/2 (B^R-B^L) normal
-# OBS! 1) "weak" formulation of split DG already includes the contribution -1/2(phi^L B^L normal)
-#         so this routine only adds 1/2(phi^L B^R normal)
-#         analogously for the Galilean nonconservative term
-#      2) this is non-unique along an interface! normal direction is super important
-function noncons_interface_flux(u_left, u_right, orientation, equation::IdealGlmMhdEquations2D)
+"""
+    noncons_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)
+    
+Strong form of nonconservative flux on a side (Powell and GLM terms)
+     phi^L 1/2 (B^L+B^R) normal - phi^L B^L normal = phi^L 1/2 (B^R-B^L) normal
+OBS! 1) The non-conservative interface flux depends on the discretization. Following "modes" are available:
+        * 1: 'weak' formulation of split DG already includes the contribution -1/2(phi^L B^L normal)
+             so this mode only adds 1/2(phi^L B^R normal)
+             ... analogously for the Galilean nonconservative term
+        * 2: This mode adds the whole non-conservative term: phi^L 1/2 (B^R-B^L)
+        * 3: This mode adds the split-form DG volume integral contribution: This is equivalent to (2)-(1)
+             - 1/2(phi^L B^L)
+     2) this is non-unique along an interface! normal direction is super important
+"""
+function noncons_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)
```

Comment:
```suggestion
@inline function noncons_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)
```
This improves the performance of `rhs!` for `examples/2d/parameters_rotor.toml` as in your test script by ca. 5% on my system (running the benchmark multiple times, this improvement is significantly bigger than fluctuations).

Reply:
Amazing!